src/main/java/chess/dto/GameDeleteResponseDto.java

```diff
@@ -0,0 +1,22 @@
+package chess.dto;
+
+public class GameDeleteResponseDto {
+    private boolean isDelete;
+    private String message;
+
+    public GameDeleteResponseDto() {
```

Comment:
```suggestion
    private GameDeleteResponseDto() {
```

setter를 추가하거나 기본 생성자를 막는게 좋겠네요.

Reply:
기본생성자를 삭제했습니다!

질문) 
기본 생성자를 삭제하지 말고 private으로 막아야하나요? 의도를 제가 잘못 파악한건가 싶어서 여쭤봅니다.

Reply:
기본생성자를 유지하려면 setter를 추가하시고, 기본생성자를 유지 하지 않으려면 private으로 막자는 뜻이었습니다.